utils/appliance/__init__.py

```diff
@@ -163,6 +163,15 @@ def get(self, cls, *args, **kwargs):
         assert 'appliance' not in kwargs
         return cls(appliance=self, *args, **kwargs)
 
+    def postgres_version(self):
+        # postgres's version is in the service name and file paths when we pull it from SCL,
+        # so this is a little resolver to help keep the version picking centralized
+        return version.pick({
+            version.LOWEST: 'postgresql92',
+            '5.5': 'rh-postgresql94',
+            '5.7': 'rh-postgresql95'
+        })
+
```

Comment:
You don't need to rely on utils.version anymore. Just use the appliances version `self.version`. In addition, you gotta change aaaaaaaall the places that do `from utils.db import postgres_version` to not do that anymore and do `appliance.postgres_version` or `current_appliance.postgres_version` depending if it in a test or not... whatever makes sense.